crates/core/tedge_mapper/src/aws/converter.rs

```diff
@@ -0,0 +1,207 @@
+use crate::core::{converter::*, error::*, size_threshold::SizeThreshold};
+
+use async_trait::async_trait;
+use clock::Clock;
+use mqtt_channel::{Message, TopicFilter};
+use thin_edge_json::serialize::ThinEdgeJsonSerializer;
+
+pub struct AwsConverter {
+    pub(crate) add_timestamp: bool,
+    pub(crate) clock: Box<dyn Clock>,
+    pub(crate) size_threshold: SizeThreshold,
+    pub(crate) mapper_config: MapperConfig,
+}
+
+impl AwsConverter {
+    pub fn new(add_timestamp: bool, clock: Box<dyn Clock>, size_threshold: SizeThreshold) -> Self {
+        let mapper_config = MapperConfig {
+            in_topic_filter: Self::in_topic_filter(),
+            out_topic: make_valid_topic_or_panic("aws/messages/monitoring"),
```

Comment:
I'm surprised to see yet another specific topic suffix here.

Reply:
The `monitoring` sub-topic is currently used for measurements, however as mentioned any suggestions on improving topic structures are appreciated.